### PR TITLE
[REVIEW] Update sink to kafka function in cybert streamz workflow

### PR DESCRIPTION
**Pull request by brhodes10**
- Updating the `sink_to_kafka` function to handle larger output messages
- Small fix to entry script parameters

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by efajardo-nv** [comment]
rerun tests

**Review by bsuryadevara — APPROVED** [review]

**Review by BartleyR — APPROVED** [review]

File: examples/streamz/scripts/entry.sh [hunk 1]
```diff
@@ -145,10 +145,10 @@ sleep 3
...
-  python -i $CYBERT_HOME/python/cybert.py --input_topic input --output_topic output --group_id $group_id --model $model_file --label_map $label_file --cuda_visible_devices $cuda_visible_devices --poll_interval $poll_interval --max_batch_size $max_batch_size --dask_scheduler ${DASK_SCHEDULER} --benchmark $benchmark
+  log "INFO" "python -i $CYBERT_HOME/python/cybert.py --input_topic $input_topic --output_topic $output_topic --group_id $group_id --model $model_file --label_map $label_file --cuda_visible_devices $cuda_visible_devices --poll_interval $poll_interval --max_batch_size $max_batch_size --dask_scheduler ${DASK_SCHEDULER} --benchmark $benchmark"
+  python -i $CYBERT_HOME/python/cybert.py --input_topic $input_topc --output_topic $output_topic --group_id $group_id --model $model_file --label_map $label_file --cuda_visible_devices $cuda_visible_devices --poll_interval $poll_interval --max_batch_size $max_batch_size --dask_scheduler ${DASK_SCHEDULER} --benchmark $benchmark
```
Comment (efajardo-nv) [inline examples/streamz/scripts/entry.sh:149]: typo here. should be $input_topic.

**Comment by efajardo-nv** [comment]
need to merge latest and resolve merge conflicts. clx streamz getting started pr was recently merged which has updates to clx streamz dockerfile and entry.sh.

File: examples/streamz/scripts/entry.sh (hunk 1 above)
Comment (brhodes10) [reply examples/streamz/scripts/entry.sh]: thx updated

**Review by efajardo-nv — APPROVED** [review]

**Pull request merged**